docs/draft/sticker-pack.md

```diff
@@ -0,0 +1,77 @@
+---
+title: IPFS gateway for Sticker Pack
+version: 0.1.0
+status: Draft
+authors: Gheorghe Pinzaru <gheorghe@status.im>
+redirect_from:
+  - /sticker-pack.html
+---
+
+## Table of Contents
+
+ 1. [Abstract](#abstract)
+ 2. [Specification](#specification)
+ 3. [Copyright](#copyright)
+
+## Abstract
+
+In this specification, we describe how Status app uses IPFS gateway to store stickers.
+We will explore image format, how they are uploaded and how an end user can see them
+inside the Status app.
```

Comment:
Does it make sense to refer to PAYLOADS spec somewhere for how stickers are sent?